onyo/cli/get.py

```diff
@@ -115,6 +115,18 @@
             One can sort by keys that are not in the output.
         """
     ),
+
+    'types': dict(
+        args=('-t', '--types'),
+        metavar="TYPES",
+        nargs='+',
+        choices=('assets', 'directories', 'templates'),
+        default=["assets"],
+        help=r"""
+            Item types to query.
+            Equivalent to onyo.is.asset=True, onyo.is.directory=True, and onyo.is.template=True.
```

Comment:
I would like these to be in the double backticks. They rendor correctly (including on the CLI).